pkg/virt-launcher/virtwrap/device/sriov/pcipool_test.go

```diff
@@ -57,13 +57,15 @@ var _ = Describe("SRIOV PCI address pool", func() {
 	})
 
 	It("provides 1 address given 1xInterface, 1xResource, 1xPCI", func() {
-		net := newNetworkData("net1", newResourceData("resource1", "0000:81:01.0"))
+		// The comma at the tail of the PCI address is intentional, validating it is ignored by the implementation.
```

Comment:
I am not sure about this, but it seems weird for me to *test* if there is a trailing comma.
Maybe checking the function as a whole, doing what it should, is Enough.

Reply:
It tests that `pciAddrString = strings.TrimSuffix(pciAddrString, ",")` does what the author intended it to do.

I am unsure what is your suggestion here.

Reply:
My suggestion is to remove the inline comment and the ` ...+"," ` part.
Testing that we successfully removed a comma from the end of a string seems redundant to me.

Reply:
It tests that we know to ignore it.
If we do not process it well, we will end up with an empty address in the pool.